utils/land-patterns.stanza

```diff
@@ -976,3 +1110,159 @@ public pcb-landpattern smd-testpoint-pkg (d:Double) :
+  make-ipc-compliant-two-pin-landpattern(
+    length, width, lead-length, false, 
+    density-level, polarized?
+  )
+
+public pcb-landpattern ipc-compliant-two-pin-landpattern (length:Toleranced,
+                                                          width:Toleranced,
+                                                          lead-length:Toleranced,
+                                                          lead-width?:False|Toleranced,
+                                                          fabrication-tolerance:Double,
+                                                          placement-tolerance:Double
+                                                          density-level:DensityLevel,
+                                                          polarized?:True|False,
+                                                          solder-mask-defined?:True|False):
+  make-ipc-compliant-two-pin-landpattern(
+    length, width, lead-length, lead-width?,
+    fabrication-tolerance, placement-tolerance, 
+    density-level, polarized?
+    solder-mask-defined?
+  )
+
+public defn ipc-compliant-two-pin-landpattern (length:Toleranced,
+                                               width:Toleranced,
+                                               lead-length:Toleranced,
+                                               lead-width?:False|Toleranced,
+                                               density-level:DensityLevel,
+                                               polarized?:True|False):
+  ipc-compliant-two-pin-landpattern(
+    length, width, lead-length, lead-width?, 
+    FABRICATION-TOLERANCE, PLACEMENT-TOLERANCE, 
+    density-level, polarized?,
+    false
+  )
+
+public defn ipc-compliant-two-pin-landpattern (length:Toleranced,       ; L
+                                                width:Toleranced,       ; W
+                                                lead-length:Toleranced, ; T
+                                                density-level:DensityLevel,
+                                                polarized?:True|False):
+  ipc-compliant-two-pin-landpattern(
+    length, width, lead-length, false, 
+    density-level, polarized?
+  )
+
+public defn chip-resistor-landpattern (part-name:String, imperial?:True|False, density-level:DensityLevel):
+  val dimensions = resistor-dimensions(part-name, imperial?)
+  ipc-compliant-two-pin-landpattern(length(dimensions)
+                                    width(dimensions)
+                                    lead-length(dimensions)
+                                    density-level
+                                    false)
+
+public defn chip-resistor-landpattern (part-name:String, density-level:DensityLeve):
+  if suffix?(part-name, "i"):
+    chip-resistor-landpattern(part-name[0 to length(part-name) - 1], true, density-level:DensityLeve)
+  else:
+    chip-resistor-landpattern(part-name, false, density-level:DensityLeve)
```

Comment:
`DensityLeve` typo.

Reply:
removed, this code referred to other things not yet ready to merge.